docs/content/en/install/gke/_index.md

```diff
@@ -43,16 +43,60 @@ On this infrastructure the K8ssandra installation will consist of the following
 Feel free to update the parameters used during this guide to match your target deployment. This should be considered a minimum for production workloads.
 
 {{% alert title="Quotas" color="primary" %}}
-This installation slightly exceeds the default quotas provided within a new project. Consider requesting the following quota requests to allow for the provisioning of this installation.
+This installation slightly exceeds the default quotas provided within a new project. Consider requesting the following quota requests to allow for the provisioning of this installation:
 
 * 6x Backend services
+
+For more, see this [troubleshooting tip]({{< relref "tasks/troubleshoot/#check-quotas" >}}).
+{{% /alert %}}
+
+## Infrastructure and Cassandra recommendations
+
+While the section above includes infrastructure settings for **minimum** production workloads, performance benchmarks reveal a wider range of recommendations that are important to consider. The performance benchmark report, available in this [blog](https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/), compared throughput and latency of:
+
+* The baseline performance of a Cassandra cluster running on AWS EC2 instances -- a common setup for enterprises operating Cassandra clusters
+* With the performance of K8ssandra running on AWS, EKS, and GCP GKE
+
+It's important to note the following additional infrastructure recommendations from the benchmark:
+
+* 8 to 16 vCPUs 
+  * r5 instances: Intel Xeon Platinum 8000 series
```

Comment:
Should we mention AWS instance types in a GKE doc?

Reply:
Oh, right. Thanks @adejanovski. Which instance type should I list for a GKE cluster in production that we'd suggest users consider?

Reply:
See updated text, thanks.